### PR TITLE
fix: make declinedBy a public field

### PR DESCRIPTION
**Pull request by LomyW**

### CODE REVIEW

File: graphql/authorizations.ts [hunk 1]
```diff
@@ -428,21 +428,21 @@ export const authorizationModelEnhanceMap: ModelsEnhanceMap = {
...
-                match: adminOrOwner,
-                matchId: adminOrOwner,
-                declinedBy: adminOrOwner,
-                participantIds: adminOrOwner,
-                organizerIds: adminOrOwner,
-            }
-        ),
+        fields: withPublicFields<
+            Lecture,
+            'id' | 'start' | 'duration' | 'createdAt' | 'updatedAt' | 'title' | 'description' | 'appointmentType' | 'isCanceled' | 'declinedBy'
```
Comment (Jonasdoubleyou) [inline graphql/authorizations.ts:433]: Why is participantIds private but declinedBy is not?

Comment (LomyW) [reply graphql/authorizations.ts:433]: For the `AttendeesModal` in the appointment detail view we need the names of the attendees (field resolver for participants and organizers) and also the information, who has declined (declinedBy array from DB), to show it in the modal.

Comment (timidak) [reply graphql/authorizations.ts:433]: True, that was my bad, @Jonasdoubleyou! 
Is it possible to check `APPOINTMENT_PARTICPANT` without a custom field resolver?
So my question is can we annotate the auto generated resolvers with `@AuthorizedDeferred`?

Comment (Jonasdoubleyou) [reply graphql/authorizations.ts:433]: Yes, @LomyW was faster ;) #622

**Pull request merged**